### ISSUE TITLE
Is it possible to have a pre-built framework?

### ISSUE THREAD

**Issue by yeahdongcn** [twelve17/openalpr-ios#4]
I came across a few problems when trying to build the framework since last night.
1. the script name changed and some of the paths are hard coded
2. couple of warnings when integrating pods
3. lib search paths changed
4. runtime_data is missing (I imported the runtime_data folder, is it right?)
I didn't know whether I'm doing it right but finally I got the the .framework generated. I'm wondering whether I could get a pre-build framework, then I can get started quickly.
Thank you for this detailed integration document.

**Comment by LordIllidan** [comment]
FYI I had the same problem - I decided to revert the git to an earlier commit, specifically this: https://github.com/twelve17/openalpr-ios/commit/ea83c3d06fbbede7856ca8f1e02d691b2777bfda

I was then able to follow the original instructions and managed to get it built and I could also detect number plates.

**Comment by yeahdongcn** [comment]
Great, I will try this tomorrow.

**Comment by fopina** [comment]
After using commit mentioned by LordIllidan everything worked perfectly. The only extra step was to manually download opencv pod as the link used by `pod install` returned 404.

I simply executed

    /usr/bin/curl -f -L -o Pods/OpenCV/file.zip "http://sourceforge.net/projects/opencvlibrary/files/opencv-ios/2.4.9/opencv2.framework.zip" --create-dirs

and only then `pod install`

**Comment by steffex** [comment]
@fopina the fix is easy, remove the file called `Podfile.lock` and change the opencv pod to: `pod 'OpenCV', '~> 2.4.9.1'`.

this will make pod install work like it should!

**Comment by twelve17** [comment]
For v2 of this project, I have replaced the old instructions and bash scripts with a Ruby script to build the OpenALPR and dependency Frameworks, as well as the OpenALPR Xcode project.  A sample app is included.  These changes have been pushed to the master branch, and the old version has been moved to branch `1_0`.  Feel free to check out this new script and see if it works for you.  I will no longer be working on or supporting the `1_0` codebase.
